spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::BooksController, type: :controller do
```

Comment:
Fill this with the tests. Hint: test for response status, it has to return 200. Test the amount of books returned, and the keys that are returned